README.md

```diff
@@ -1,50 +1,39 @@
+byoLAD augments your development experience by reviewing your code, identifying errors, adding documentation, and suggesting general improvements.  Since byoLAD is open-source, you can modify prompts and other configurations to your liking. 
 
-## Features
-
-This AI platform is an open-source AI extension. Some of the main features are code review, code completion, and code discussion. The extension also allows for any type of API to be used. You can use whatever AI API you are currently using or comfortable with. 
-
-### Code Review
-
-This extension has the option of reviewing a whole file or reviewing selected code. When it reviews your code, it will look for bugs in the code and look for general naming problems. In addition, it will suggest getting rid of or adding spacing to make the code easier to read. When you ask it to review your code, it will open another tab showing a dif view of your code and the code that it suggests. You can then choose which changes you will add to your code. 
-
-### Code Completion
+In addition, byoLAD enables you to use your own LLM API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
+#### LLM Providers Currently Supported: 
+- OpenAI
+- Google
 
-This extension will help as you type to complete the code for you. With the tool, you can have it generate functions and other snippets of code. It will analyze what you have written so far, and it will suggest what it thinks you want. 
+![](ReviewCode.gif)
 
-### Code Discussion
-
-This extension also allows you to discuss or modify the generated code. Instead of trying to find the right prompt to get the code you are looking for, you can suggest changes to the proposed code until you have the code you want. 
-
-## Requirements
-
-The only requirement needed if not running locally would be to set your API key in the setting, which is explained in this document. You will have to get your own API key for the AI service you would like to use like OpenAI or ChatGPT. 
+## Features
 
-<a target="_blank" href="https://code.visualstudio.com/docs/getstarted/settings">How to set your API key</a>
+- Option to review a whole file or just a piece of selected code
+- Formatting suggestions like getting rid of extra lines to make it easier to read
+- Identifying bugs in code
+- Adds Documentation
+- Checks Spelling
+- General improvement suggestions
```

Comment:
Standardize the verb tenses to 3rd person present (e.g. "Suggests formatting improvements", "Identifies bugs", etc)